### ISSUE TITLE
Using in jsx

### ISSUE THREAD

**Issue by iBreadInside** [iiroj/use-breakpoint#11]
Hello! I'm trying to use that library in my react project, but recieve this error. Can you help me to fix it?
![image](https://user-images.githubusercontent.com/66948071/148520092-0409f15a-0fb6-4745-b6aa-b0b9cab8f102.png)

**Comment by iiroj** [comment]
Thanks for the report, this seems to be a problem with how `tsc` compiles the JS files. It should emit

```js
import useBreakpoint from './useBreakpoint.js';
```

instead of

```js
import useBreakpoint from './useBreakpoint';
```

**Comment by iiroj** [comment]
This seems related: https://github.com/microsoft/TypeScript/issues/16577

**Comment by iiroj** [comment]
This should be fixed in version `3.0.1`!
